docs/API-functions.md

```diff
@@ -578,8 +578,24 @@ setting the `SRTO_RENDEZVOUS` option to true, and doing `srt_connect`.
 * `local_name`: specifies the local network interface and port to bind
 * `remote_name`: specifies the remote party's IP address and port
 
+- Returns:
+
+  * `SRT_ERROR` (-1) in case of error, otherwise 0
+
+- Errors:
+
+  * `SRT_EINVSOCK`: Socket passed as `u` designates no valid socket
+  * `SRT_EINVOP`: Socket already bound
+  * `SRT_ECONNSETUP`: Internal creation of a UDP socket failed
+  * `SRT_ESOCKFAIL`: Internal configuration of a UDP socket (`bind`, `setsockopt`) failed
+  * `SRT_ECONNSOCK`: Socket `u` is already connected
+  * `SRT_ECONNREJ`: Connection has been rejected
+
 **NOTE:** The port value shall be the same in `local_name` and `remote_name`.
 
+IMPORTANT: It's not allowed to bind and connect the same socket to two
+different families (that is, both `source` and `target` must be `AF_INET` or
+`AF_INET6`), although you may mix links over IPv4 and IPv6 in one group.
```

Comment:
The last sentence is not necessary because rendezvous mode for group connections is not supported, and it's hard to expect that it will be anyhow in future.

Reply:
If it is ok with you, I will change to:
`IMPORTANT: It's not allowed to perform a rendezvous connection to two
different families (that is, both `local_name` and `remote_name` must be `AF_INET` or
`AF_INET6`).`

Reply:
Yes, that's ok.